.github/workflows/cli-assets-model-job-model-as-input.yml

```diff
@@ -45,5 +45,5 @@ jobs:
       run: |
           source "${{ github.workspace }}/infra/sdk_helpers.sh";
           source "${{ github.workspace }}/infra/init_environment.sh";
-          az ml model create -f assets/model/job-model-as-input.yml
```

Comment:
Please change the name of the yml file from [job-model-as-input.yml](https://github.com/Azure/azureml-examples/blob/main/cli/assets/model/job-model-as-input.yml) to [model-job-as-input.yml](https://github.com/Azure/azureml-examples/blob/main/cli/assets/model/job-model-as-input.yml)

The workflows are generator is picking the noun from the second part of the file name.
https://github.com/Azure/azureml-examples/blob/main/cli/readme.py#L510

Reply:
Also, Delete both these workflows:
.github/workflows/cli-assets-model-job-model-as-input.yml
.github/workflows/cli-assets-model-job-model-as-output.yml

and regenerate the workflows again using the command: `python readme.py` from CLI directory.